elasticdl/docker/scripts/install-go.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+curl --silent https://dl.google.com/go/go1.13.4.linux-amd64.tar.gz | tar -C /usr/local -xzf -
+
+go get github.com/golang/protobuf/protoc-gen-go
```

Comment:
Are all these required? Maybe we can add some of them later on when they become necessary

Reply:
I copied it from sqlflow.
golint, goyacc etc are used for pre-commit check of the go files.
gotest, goveralls are needed for testing.